app/models/team_member.rb

```diff
@@ -66,27 +65,25 @@ def team_status
     end
   end
 
-  def status
-    if team == Team.board
-      BOARD_STATUS
-    else
-      team_status
-    end
+  def board_role
```

Comment:
Actually, Board members are not yet migrated, only the Board team is migrated to Board group. Board members will be migrated in a later PR because it's slightly complicated due to plenty of unit tests.